lib/components/serverside-data/ServersideTable.vue

```diff
@@ -1,7 +1,22 @@
 <template>
   <ServersideIterator ref="iterator" v-bind="$attrs" :loading.sync="loading">
     <template #items="{ items }">
-      <Table v-bind="{ ...tableAttrs, items, icon, loading }">
+      <CustomTable
+        v-if="customTable"
+        v-bind="{ ...tableAttrs, items, icon, loading }"
+        :server-items-length="items.length"
```

Comment:
This is not the actual server items length. You have to pass the total items in the database. Maybe https://vuetifyjs.com/en/api/v-data-table/#props helps in order to disable the client side ordering.

Reply:
I now use the `count` property from `ServersideTable`.